docs/reference/environment.md

```diff
@@ -0,0 +1,16 @@
+# Environment Variables
+
+This page lists the environment variables available for Shift to serve as a guide to set the a custom environment for Shift.
+
+| Environment Variable | Description |
+| :--- | :---------- | 
+| **SHIFT_PATH_WORKFLOWS** | Directory paths set in this variable will be considered as workflow project locations. This means that workflows existing in these directories will use relative paths when being referenced in [Workflow](nodes/workflow#workflow-node) and [WorkflowProcess](nodes/workflow#workflowprocess-node) nodes.|
+| **SHIFT_ACTIVATION_SCRIPT** | This variable should to an activation script to be run before a workflow is executed in  [Batch](../getting_started/basics/batch.md) mode. 

```

Comment:
@juliavisserinbibo as I updated in this comment https://github.com/Inbibo/shift/pull/1115#discussion_r1738545212 This environment variable should not be documented.

Reply:
Done